client/watch.js

```diff
@@ -1036,6 +1069,22 @@ Template.context_browser.helpers({
   mediaTypeForDisplay (){
     return pluralizeMediaType(Session.get('mediaDataType') || Session.get('previousMediaDataType')).toUpperCase();
   },
+  replayAvailable(){
+    console.log(mainPlayer.activeStream.get());
+    
+    if(mainPlayer.activeStream.get().source === "youtube" && !mainPlayer.activeStream.get().live){
```

Comment:
Seems to work for me, switching between a live/not live stream.  `mainPlayer.activeStream` should be being set by the Tracker.autorun on line 398 of watch.js